eclass/kernel-install.eclass

```diff
@@ -79,7 +79,7 @@ _IDEPEND_BASE="
 
 LICENSE="GPL-2"
 if [[ ${KERNEL_IUSE_GENERIC_UKI} ]]; then
-	IUSE+=" generic-uki"
+	IUSE+=" generic-uki module-compress"
```

Comment:
We need to RDEPEND on `sys-apps/kmod[lzma]` for this to be able to load the modules.